.palantir/revapi.yml

```diff
@@ -67,9 +67,45 @@ acceptedBreaks:
       justification: "Removing deprecated method"
   "1.1.0":
     org.apache.iceberg:iceberg-core:
+    - code: "java.class.noLongerImplementsInterface"
+      old: "class org.apache.iceberg.jdbc.JdbcCatalog"
+      new: "class org.apache.iceberg.jdbc.JdbcCatalog"
+      justification: "Needed to remove compilation dependency on Hadoop"
```

Comment:
I think the justification should be that this is a safe change because of type erasure.